src/cr/cube/crunch_cube.py

```diff
@@ -1059,6 +1059,33 @@ def percentages(self, axis=None):
         '''
         return self.proportions(axis) * 100
 
+    def population_counts(self, population_size):
+        '''Get the population counts for crunch cube values relative to the total population size estimate.
+
+        This function calculates the population counts for crunch cube values. The
+        population counts are based on the values of the 'proportions'.
+
+        Args
+            population_size (int): Estimated total population size
+
+        Returns
+            (nparray): Calculated array of crunch cube population counts.
+
+        Example:
+            >>> cube = CrunchCube(fixt_cat_x_cat)
+            np.array([
+               [5, 2],
+               [5, 3],
+            ])
+
+            >>> cube.population_counts(9000)
+            np.array([
+                [3000, 1200],
+                [3000, 1800],
+            ])
+        '''
+        return self.proportions() * population_size
```

Comment:
Forgive my ignorance (about population counts), but is it _certain_ that we don't need to be dealing with axes here (like direction of the proportions)?
@kwypchlo :point_up:

Reply:
yup, that's my understanding and this is how it works in whaam

Reply:
@slobodan-ilic is right, these should be proportions *in some direction* multiplied by [filtered] population number

Reply:
but it's actually an estimate of the count in the cell, which doesn't change, it would be with respect to the whole table